src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor({ health = 100, name }) {
```

Comment:
The constructor of the Animal class should take `name` as a string, not an object. The current implementation will result in `undefined` for the `name` property because it's trying to destructure a string.